### ISSUE TITLE
Support GQL `tokens` endpoint

### ISSUE THREAD

**Issue by trevorbernard** [Granola-Team/mina-indexer#1258]
Tokens list is a addresses page for listing all the tokens. It will require the following columns

* Token Name (searchable)
* Token ID (searchable)
* Supply
* Token Owner
* Token Holders (count)
* Total Transactions (count)
* Locked Status

**Comment by trevorbernard** [comment]
This is blocked by: https://github.com/Granola-Team/mina-indexer/issues/1257

**Comment by Isaac-DeFrain** [comment]
#1684 partially implements this: we can now search by token id/address

**Comment by Isaac-DeFrain** [comment]
I don't think this issue is blocked anymore

**Comment by jhult** [comment]
Related to Granola-Team/mina-block-explorer#645

**Comment by Isaac-DeFrain** [comment]
Blocked by #869

**Comment by Isaac-DeFrain** [comment]
https://github.com/Granola-Team/mina-indexer/commit/c9580ebf62ffc87fff3830bded6aa43ebbe05b05 .. https://github.com/Granola-Team/mina-indexer/commit/83f5050ccf8d88f8fdb1d8e66cf30e68c01f1b3a

Will potentially need to add transaction data. Otherwise done. Closing in favor of #1786
